INSTALL.md

```diff
@@ -40,7 +40,19 @@ The [libbpf](https://github.com/libbpf/libbpf) is a dependency of CNDP. Starting
 it can be installed using apt. For earlier Ubuntu versions, or for users who want the latest code,
 it can be installed from source.
 
-### Install libbpf from package manager
+#### _Note:_
+
+```bash
+Newer versions of libbpf greater than or equal to v0.7.0 require _libxdp_ to be installed. For now we
+can checkout a previous version v0.5.0 or v0.6.1 instead of installing _libxdp_.
+```
+
+### Install libbpf-dev from package manager
+
+Installing **libbpf-dev** installs the header files required to build CNDP.
```

Comment:
This sentence and the next one are redundant. This whole paragraph is oddly worded, and the line length is long.

```
Use on the following command on Ubuntu 20.10 and later to install the headers and libraries to build
and run CNDP applications. If using an earlier Ubuntu version, you need to build libbpf from source.
```